types/hand-crafted/modules/RTC/JitsiTrack.d.ts

```diff
@@ -0,0 +1,28 @@
+import JitsiConference from '../../JitsiConference';
+import { MediaType } from '../../service/RTC/MediaType';
+import TraceablePeerConnection from './TraceablePeerConnection';
+
+export default class JitsiTrack {
```

Comment:
Might need to be extending EventEmitter from `event` for addListener() and removeListener()
The constructor also aliases these as `addEventListener()` and `removeEventListener()`.

Can't seem to get this recognised locally though

Reply:
Will be interesting if we can persuade the jsdocs auto-generated to do the same